src/main/java/org/beanpod/switchboard/entity/LogEntity.java

```diff
@@ -29,6 +32,17 @@
 
   private String level;
 
-  private String
-      serialNumber; // This can be a single serialNumber or two serialNumbers seperated by a comma
+  // first device
+  private String serialNumber;
+
+  @OneToOne(mappedBy = "logEntity", cascade = CascadeType.ALL)
+  @PrimaryKeyJoinColumn
+  private StreamLog streamLog;
+
+  public LogEntity(OffsetDateTime dateTime, String message, String level, String serialNumber) {
+    this.dateTime = dateTime;
+    this.message = message;
+    this.level = level;
+    this.serialNumber = serialNumber;
+  }
```

Comment:
whats this for?

Reply:
instead of using a constructor, you can just use the lombok builder

Reply:
I removed it and used @RequiredArgsConstructor